### PR TITLE
Fix duplicate value in consumesMimeTypesMetadata

### PR DESCRIPTION
**Pull request by rtaylor-gci**
This fixes a bug where the Adapter reports incorrect values in Capabilities.ConsumesMimeTypes, resulting in Harbor being unable to scan OCI images.  Related adapter metadata tests have also been added.  This resolves Carbon Black Case Number 00318433.

Tested locally in our container platform.

I have not updated any app or Chart version numbers under the assumption that the internal team would like to reserve that operation for themselves.

### CODE REVIEW

**Review by filas-cb — APPROVED** [review]
Changes look good

**Comment by rtaylor-gci** [comment]
Thanks, @filas-cb.  I'm still waiting for our legal team to approve the CCLA.  I'm guessing this will remain blocked until then?

**Comment by filas-cb** [comment]
> Thanks, @filas-cb. I'm still waiting for our legal team to approve the CCLA. I'm guessing this will remain blocked until then?

I'm actually not familiar with the flow in this context, I can only assume merging will remain blocked until then.

**Comment by filas-cb** [comment]
@rtaylor-gci  wondering if you could approve my PR (where I simply cloned your changes) in
https://github.com/vmware/carbon-black-adapter-for-harbor/pull/25

I'm afraid I may hot the same CLA snag

**Comment by rtaylor-gci** [comment]
@filas-cb I'm not sure how I can help here?  I can write a review on your PR, but I don't have write access to this repo so it won't satisfy the merge requirement.

**Pull request closed without merging**
